js/form/src/Dropdown.svelte

```diff
@@ -25,6 +25,10 @@
 		showOptions = false,
 		filterInput: HTMLElement;
 
+	if (typeof value === "string") {
```

Comment:
If the problem is that non-string values can't be displayed in the drop-down, then I think it would be better to do the coercion in the postprocess method so that even if someone accidentally updates the value to be a non-string after the app launches then we will be covered too.

And I think at that point, we can get rid of this `if` on the front-end since we're guaranteeing that the value will be string so we can always set inputValue=value on init? 


Also think it would be ok to raise an error on the backend would be ok in this case.

Reply:
Actually the reason for this is to exclude the cases of `multiselect=True` (where value would be a list of strings) or if it's null. The inputValue is only set in the case where there is a value and it is a single string, and this is the simplest way to do it to also pass type checks. This isn't a guard against improper types being passed by a user.

Reply:
Gotcha!